tedge/src/certificate.rs

```diff
@@ -43,25 +41,31 @@ pub struct CreateCertCmd {
     id: String,
 
     /// The path where the device certificate will be stored
-    cert_path: String,
+    cert_path: FilePath,
 
     /// The path where the device private key will be stored
-    key_path: String,
+    key_path: FilePath,
+
+    /// The config repository for updating the TEdgeConfig
+    config_repository: TEdgeConfigRepository,
 }
 
 /// Show the device certificate, if any
 pub struct ShowCertCmd {
     /// The path where the device certificate will be stored
-    cert_path: String,
+    cert_path: FilePath,
 }
 
 /// Remove the device certificate
 pub struct RemoveCertCmd {
     /// The path of the certificate to be removed
-    cert_path: String,
+    cert_path: FilePath,
 
     /// The path of the private key to be removed
-    key_path: String,
+    key_path: FilePath,
+
+    /// The config repository for updating the TEdgeConfig
+    config_repository: TEdgeConfigRepository,
```

Comment:
Ok for this PR. The next step is to remove it, implementing https://github.com/thin-edge/thin-edge.io/issues/86.